packages/manager/modules/pci/src/projects/project/storages/containers/containers.service.js

```diff
@@ -433,6 +436,24 @@ export default class PciStoragesContainersService {
     ).$promise;
   }
 
+  updateContainer(projectId, containerModel) {
+    const { region, name, versioning, s3StorageType } = containerModel;
+
+    const url =
+      s3StorageType === 'storage'
+        ? `/cloud/project/${projectId}/region/${region}/storage/${name}`
+        : `/cloud/project/${projectId}/region/${region}/storageStandard/${name}`;
+
+    return this.$http
+      .put(url, {
+        versioning,
+      })
+      .then(({ data }) => data)
+      .then(() => {
+        return this.$q.resolve();
+      });
```

Comment:
becuase when the modal is closed we go back to the details page and the call to get the detail will be executed there

Reply:
Ok so, the best practice would therefore be to ignore the response here, not intentionally discard it. This can lead to potential side effects. We could use the response later